src/front/app/features/create/components/CreateError.js

```diff
@@ -0,0 +1,33 @@
+import React, { PropTypes } from 'react';
+
+const CreateError = ({ formFields }) => {
+    let errorMessage = '';
+
+    Object.keys(formFields).forEach((key) => {
+        if (formFields[key].error && formFields[key].touched) {
+            errorMessage = 'Fill all necessary fields';
+
+            return;
+        }
+    });
+
+    return (
+        <div className={(errorMessage === '') ?
+          'create-error create-error_hide' : 'create-error'}
+        >
+
+            <div className="create-error__message">
+                <div className="create-error__holder"></div>
+                {errorMessage}
```

Comment:
I think, it is better keep text in html for now and use conditions to show or hide it. Test is not changing, you just show of hide it, so logic should be appropriate.